ibis/backends/duckdb/__init__.py

```diff
@@ -584,6 +591,22 @@ def to_parquet(
             DuckDB Parquet writer arguments. See
             https://duckdb.org/docs/data/parquet#writing-to-parquet-files for
             details
+
+        Examples
+        --------
+        Write out an expression to a single parquet file.
+
+        >>> import ibis
+        >>> penguins = ibis.examples.penguins.fetch()
+        >>> con = ibis.get_backend(penguins)
+        >>> con.to_parquet(penguins, "penguins.parquet")
+
+        Write out an expression to a hive-partitioned parquet file.
+
+        >>> import ibis
+        >>> penguins = ibis.examples.penguins.fetch()
+        >>> con = ibis.get_backend(penguins)
+        >>> con.to_parquet(penguins, "penguins_hive_dir", partition_by=("year"))
```

Comment:
Are the parens around `"year"` intentional?

Reply:
yes-ish?  They're required if you partition on more than one column, but aren't required for a single-column

Reply:
I'll add examples of both

Reply:
Hm? Does it need a comma?

```
>>> ("year") == "year"
True
```

Reply:
no, `duckdb` barfs if you pass in a 1-tuple